guides/common/modules/proc_importing-kickstart-repositories.adoc

```diff
@@ -28,7 +28,7 @@ endif::[]
 ----
 
 ifeval::["{rhel-version}" == "8"]
-. Create directories for {RHEL}Linux 8 AppStream and BaseOS Kickstart repositories:
+. Create directories for {RHEL} 8 AppStream and BaseOS Kickstart repositories:
```

Comment:
```suggestion
. Create directories for {RHEL} {rhel-version} AppStream and BaseOS Kickstart repositories:
```
Would it work/make sense to use the attribute instead?

Reply:
I'm not sure. This whole section is very version specific. I'd rather not get into this. This really was a random fix that I happened to run into.